invisible_cities/cities/detsim_simulate_photons.py

```diff
@@ -0,0 +1,44 @@
+import numpy  as np
+
+from typing import Callable
+
+
+#######################################
+########## PHOTON SIMULATION ##########
+#######################################
+def generate_s1_photons(energies : np.array,
+                        ws       : float) -> np.array:
+    """generate s1 photons"""
+    return np.random.poisson(energies / ws)
+
+
+def generate_s2_photons(x              : np.array,
+                        el_gain        : float,
+                        el_gain_sigma  : float) -> np.array:
+    """generate number of EL-photons produced by secondary electrons that reach
+    the EL (after drift and diffusion)
+    """
+    n = len(x)
+    nphs = np.random.normal(el_gain, el_gain_sigma, size = n)
+    return nphs
```

Comment:
x is an array with the size of the number of secondary electrons that arrive to the EL, so any of the arrays from the output of `diffuse_electrons` in PR #695. The reason why this is implemented in that way is because in the dataflow dictionary (PR #700) just the arrays appear. In my opinion this must be modified, so suggestions are welcome

Reply:
I think 'x' is in this case a bad name, maybe in this case you can pass _n_ as argument, the len of _x_ and _number_of_electrons_